### PR TITLE
Clarify tags doc is not just about Varnish

### PR DESCRIPTION
**Pull request by andrerom**

### CODE REVIEW

File: docs/using_tags.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ They work in a similar way as [persistence cache tags in eZ Platform v2](https:/
 - Like an index in a database it is typically used for anything relevant that represent the given cache item
 - Used for cache invalidation
 
-There are also other aspects to this bundle, like the fact that by using Varnish xkey over BAN we can now do soft purge,
-but more on that in [varnish/README.md](varnish/README.md) and the corresponding VCL file.
+It works across all supported proxies _(see ["drivers"](drivers.md))_ by eZ Platform:
+- Symfony Proxy _(PHP based for single server usage, primarly for smaller web sites)_
```
Comment (DominikaK) [inline docs/using_tags.md]: `primarly` -> `primarily`

File: docs/using_tags.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ They work in a similar way as [persistence cache tags in eZ Platform v2](https:/
...
+- [Varnish](https://varnish-cache.org/) with [xkey module](https://github.com/varnish/varnish-modules) _or_ [Varnish Plus](https://www.varnish-software.com/products/varnish-plus/) _(High performance reverse proxy)_
+- [Fastly](https://www.fastly.com/) _(High performance reverse proxy, orignally based on Varnish, worldwide as a CDN, available in eZ Platform Enterprise)_
 
-For a bit less abstract way of saying this; It allows us to do things like tagging every article response, and when
-article content type gets an update we can tell Varnish all articles should be considered stale so they are updated in
-the background once someone requests them. Same goes for other operations by the repository.
+Varnish and Fastly is highly recommended for medium to large traffic needs. Besides being able to handle much more traffic, supported for use in cluster setup, they also both support soft purge _(by tags)_, meaning they are able to serve stale content while it's refreshed in the background on-demand, leading to more stable load on your backend.
```
Comment (DominikaK) [inline docs/using_tags.md]: `Varnish and Fastly are...`

File: docs/using_tags.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ They work in a similar way as [persistence cache tags in eZ Platform v2](https:/
 - Like an index in a database it is typically used for anything relevant that represent the given cache item
 - Used for cache invalidation
 
-There are also other aspects to this bundle, like the fact that by using Varnish xkey over BAN we can now do soft purge,
-but more on that in [varnish/README.md](varnish/README.md) and the corresponding VCL file.
+It works across all supported proxies _(see ["drivers"](drivers.md))_ by eZ Platform:
+- Symfony Proxy _(PHP based for single server usage, primarily for smaller web sites)_
+- [Varnish](https://varnish-cache.org/) with [xkey module](https://github.com/varnish/varnish-modules) _or_ [Varnish Plus](https://www.varnish-software.com/products/varnish-plus/) _(High performance reverse proxy)_
+- [Fastly](https://www.fastly.com/) _(High performance reverse proxy, orignally based on Varnish, worldwide as a CDN, available in eZ Platform Enterprise)_
```
Comment (bdunogier) [inline docs/using_tags.md]: Typo on "originally".

File: docs/using_tags.md [hunk 4]
```diff
@@ -7,12 +7,12 @@ They work in a similar way as [persistence cache tags in eZ Platform v2](https:/
...
 
-For a bit less abstract way of saying this; It allows us to do things like tagging every article response, and when
-article content type gets an update we can tell Varnish all articles should be considered stale so they are updated in
-the background once someone requests them. Same goes for other operations by the repository.
+Varnish or Fastly are highly recommended for medium to large traffic needs. Besides being able to handle much more traffic, supported for use in cluster setup, they also both support soft purge _(by tags)_, meaning they are able to serve stale content while it's refreshed in the background on-demand, leading to more stable load on your backend.
```
Comment (bdunogier) [inline docs/using_tags.md]: I'd break that sentence, it is too long. Suggestion:

> Besides being able to handle much more traffic, supported for use in cluster setup, they also both support soft purge _(by tags)_. It enables them to serve stale content while it is refreshed in the background, leading to more stable load on your backend.

We could also add that stale cache improves the user experience by diminishing loading times.

Comment (andrerom) [reply docs/using_tags.md]: Reviews after merge here, but I don't disagree, so open PR with a wording of your liking and we can merge it.

Comment (bdunogier) [reply docs/using_tags.md]: Well, I was on vacation. If it's only "my liking", then no need to open a PR.

File: docs/using_tags.md (hunk 3 above)
Comment (glye) [reply docs/using_tags.md]: Fixed in https://github.com/ezsystems/ezplatform-http-cache/commit/3c7ff728893662e94c10224e26541f6100d96f34

**Pull request merged**